hbase-server/src/main/java/org/apache/hadoop/hbase/replication/regionserver/ReplicationSource.java

```diff
@@ -119,6 +120,14 @@
   private int logQueueWarnThreshold;
   // ReplicationEndpoint which will handle the actual replication
   private volatile ReplicationEndpoint replicationEndpoint;
+  //This is needed for the startup loop to identify when there's already
+  // an initialization happening (but not finished yet),
+  // so that it doesn't try submit another initialize thread.
+  // NOTE: this should only be set to false at the end of initialize method, prior to return.
+//  private final AtomicBoolean startupOngoing = new AtomicBoolean(false);
```

Comment:
nit: indent?

Reply:
Actually, these whole block can be removed.